website/src/docs/plugins.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "List of Plugins"
```

Comment:
I think we mostly use double quotes in YAML, actually, and single quotes around “category” everywhere are a mistake.

Reply:
:grimacing:

Reply:
https://github.com/transloadit/uppy/commit/d820cdeea5932221f5c79910975705c92f127c23